test/__snapshots__/TypeScriptGenerator-test.ts.snap

```diff
@@ -56,13 +52,15 @@ fragment FeedbackComments_feedback on Feedback {
 
 ~~~~~~~~~~ OUTPUT ~~~~~~~~~~
 // QueryWithConnectionField.graphql
-type FeedbackComments_feedback$ref = any;
+type FragmentRefs<Refs extends string> = null | {
+    [ref in Refs]: true;
+};
```

Comment:
I think we should move this into the relay-runtime DT package and thus add an import here. It will be useful to re-use for e.f. `readInlineData`.

Reply:
Btw, why is this a union including `null`?

Reply:
I undid this nullability. I put it there originally because I thought it would help creating mock data and wouldn't affect assignability, but it does affect assignability.